flow.py

```diff
@@ -72,7 +72,7 @@ def check_last(self):
         updating the NetworkSimulator object accordingly
         """
         if self.last_acknowledged == self.num_packets:
-            self.ns.num_active_flows -= 1
```

Comment:
Check to make sure the dictionary is empty not that the last packet is unacknowledged